_posts/2017-02-21-release-3-0-0.markdown

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Release 3.0.0"
+date: 2017-02-21 14:00:00
+author: "@jubianchi"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.0.0][milestone]!
+
+## Stats
+
+* 55 [commits][commits],
+* 580 files changed,
+* 8 contributors,
+* 5 [new features][changelog],
+* 1 [bug fix][changelog].
+
+Let's talk about these numbers. Especially one of them: you might have noticed the 580 files changed. This is a lot! 
+atoum has a total of 557 files (source files and test files) so 580 is really a lot of change but this is for some good 
```

Comment:
557 only counts for `classes/` and `tests/`. When applying PSR-2 we also changed some files in `resources/` for example.

We also added/changed some metadata file (README, UPGRADE, ...) and I deleted some files after we applied PSR2.

557 is the number of file on master *today*.